From 0bb555bc6f978d636685d97a2cc6e0c03a6c909c Mon Sep 17 00:00:00 2001
From: Felix <google@ix5.org>
Date: Mon, 27 May 2019 23:50:59 +0200
Subject: [PATCH] releasetools: Skip adding compatiblity.zip

Forcing HasTrebleEnabled() to False means the compatiblity.zip archive
will not be appended to the ota package.
---
 tools/releasetools/ota_from_target_files.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tools/releasetools/ota_from_target_files.py b/tools/releasetools/ota_from_target_files.py
index 10a19b368..503d89628 100755
--- a/tools/releasetools/ota_from_target_files.py
+++ b/tools/releasetools/ota_from_target_files.py
@@ -596,8 +596,7 @@ def HasVendorPartition(target_files_zip):
 
 
 def HasTrebleEnabled(target_files_zip, target_info):
-  return (HasVendorPartition(target_files_zip) and
-          target_info.GetBuildProp("ro.treble.enabled") == "true")
+  return False
 
 
 def WriteFingerprintAssertion(script, target_info, source_info):
-- 
2.21.0

